src/ToggleButton.tsx

```diff
@@ -17,7 +17,7 @@ export interface ToggleButtonProps
   disabled?: boolean;
   onChange?: React.ChangeEventHandler<HTMLInputElement>;
   value: unknown;
-  inputRef?: React.LegacyRef<'input'>;
+  inputRef?: React.LegacyRef<'input'> | React.RefObject<HTMLInputElement>;
```

Comment:
```suggestion
  inputRef?: React.LegacyRef<HTMLInputElement>;
```
I think this was just an oversight and the original typing is wrong.  The `value: unknown` right above it should be narrowed to `value: string | ReadonlyArray<string> | number;` to match that in the React types.

Reply:
Thanks. I made the update. I also realized I forgot to update the propTypes. Please make sure I did those correctly, as I've not used those before.

Also, worth noting that changing the `value` typedef from `unknown` will be a compile-time error for anyone not using a `string | number`. I had actually been using `boolean` for my value, as in my case I was toggling between 2 states (on/off), and that was the cleanest way to track the value for me. Easy enough to update to string, and map back to boolean for the consumers of my widget, but I thought I would mention it, as I'm not sure how you handle changes like that in this library.